system/preside-objects/email/email_template_send_log.cfc

```diff
@@ -56,4 +56,7 @@ component extends="preside.system.base.SystemPresideObject" {
 	property name="activity_code"       formula="activities.code";
 	property name="activity_reason"     formula="activities.reason";
 	property name="activity_date"       formula="activities.datecreated" type="date" dbtype="datetime";
+
+	property name="content_html" formula="content.html_body";
+	property name="content_text" formula="content.text_body";
```

Comment:
Properties in formula fields should always be prefixed with `${prefix}` - so, `formula="${prefix}content.html_body"`.

Reply:
Updated with prefix.